src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -88,9 +89,27 @@ public Response createItem(Item payload) throws Exception {
       return createForbidden("Anonymous users can not create items");
     }
     
-    if (payload.getPaymentMethods() == null) {
-      return createBadRequest("PaymentMethods is required");
+    Currency priceCurrency = getPriceCurrency(payload.getUnitPrice());
+    
+    if (payload.getTypeOfBusiness() == TypeOfBusinessEnum.SALE) {
+      if (payload.getPaymentMethods() == null) {
+        return createBadRequest("PaymentMethods is required");
+      }
+      
+      if (payload.getUnitPrice() == null) {
+        return createBadRequest(String.format("Price is required"));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "Price is required" 4 times. ([squid:S1192](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=47&issues=AWZVH_tLBfefbO9HrBx1&open=AWZVH_tLBfefbO9HrBx1)

[ISSUE_KEY]: AWZVH_tLBfefbO9HrBx1